functions/package.json

```diff
@@ -20,8 +20,8 @@
     "@algolia/client-common": "^4.8.6",
     "@algolia/transporter": "^4.8.6",
     "algoliasearch": "^4.8.6",
-    "firebase-admin": "^9.5.0",
-    "firebase-functions": "^3.13.2"
+    "firebase-admin": "^9.8.0",
```

Comment:
could you update the package-lock.json too?

Reply:
Done :)